File: app/models/expense.rb
class Expense < ApplicationRecord
  belongs_to :category
  belongs_to :employee

  def self.from_csv row
    new \
      date: Date.strptime(row["date"], "%m/%d/%Y"),
      description: row["expense description"],
      pretax_amount: row["pre-tax amount"].gsub(",", "_"),
      tax_name: row["tax name"],
      tax_amount: row["tax amount"].gsub(",", "_")
  end
end

File: app/models/expense.js
// Expense model

var db = require('orm').db;

var Expense = db.define('expense', {
  date: Date,
  category: String,
  employeeName: String,
  expenseDescription: String,
  preTaxAmount: Number,
  taxName: String,
  taxAmount: Number,
  sourceFileName: String,
  sourceFileID: Number
}, {
  methods: {
    example: function(){
      // return example;
    }
  }
});

Expense.sync(function (err) {
  if (!err) { console.log('Expense table loaded.')}
  else { console.error('Error loading Expense table :', err); }
});
